presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -124,9 +124,20 @@ Property                                                Description
     specific cipher suites by setting the ``http-server.https.included-cipher``
     property:
 
+    Monitor CPU usage on the Presto coordinator after enabling HTTPS. Java
+    prefers the more CPU-intensive cipher suites if you allow it to choose from
+    a big list. If the CPU usage is unacceptably high after enabling HTTPS,
+    you can configure Java to use specific cipher suites by setting
+    the ``http-server.https.included-cipher`` property to only allow
+    cheap ciphers. Non FS ciphers are disabled by default. As a result, if you
```

Comment:
Let's spell out "forward secrecy (FS)" as that might not be obvious to anyone not well versed in TLS.